bgpd/bgp_updgrp_adv.c

```diff
@@ -636,12 +644,25 @@ void subgroup_announce_table(struct update_subgroup *subgrp,
 							    rn_p, &attr))
 					bgp_adj_out_set_subgroup(rn, subgrp,
 								 &attr, ri);
-				else
+				else {
+					/* If default originate is enabled for
+					 * the peer, do not send explicit
+					 * withdraw. This will prevent deletion
+					 * of default route advertised through
+					 * default originate
+					 */
+					if (CHECK_FLAG(
+						    peer->af_flags[afi][safi],
+						    PEER_FLAG_DEFAULT_ORIGINATE)
+					    && is_default_prefix(&rn->p))
+						break;
```

Comment:
I'm just curious here: shouldn't it be `continue` here? I'm about if ECMP for 0.0.0.0/0, then isn't that case when we withdraw only a single prefix while there are more in ECMP group?

Reply:
I checked with multipath configuration using addpaths and found it is working for this case
as mentioned in logs (multipath.txt) above